search-lambda/CKS.SearchLambda/CKS.SearchLambda.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <AWSProjectType>Lambda</AWSProjectType>
+
+    <!-- This property makes the build directory similar to a publish directory and helps the AWS .NET Lambda Mock Test Tool find project dependencies. -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <NoWarn>1701;1702;NU1605</NoWarn>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Amazon.Lambda.APIGatewayEvents" Version="2.4.0" />
+    <PackageReference Include="Amazon.Lambda.Core" Version="1.2.0" />
```

Comment:
Can we use Amazon.Lambda.AspNetCoreServer instead, like we do with MAS? This means that we can run it locally in IIS express, to mimic the API gateway. This means you can do ajax requests from Gatsby like we currently do. Currently it looks like it would hard to dev the search api integration on the static site side.

Reply:
The Mock Lambda Test Tool looks great, but it doesn't allow us to hit it via ajax requests

Reply:
this will allow the integration to be tested/ran locally with gatsby running locally